completion/utilities.py

```diff
@@ -0,0 +1,18 @@
+#Python imports
+from __future__ import unicode_literals, absolute_import
+
+#Project imports
+from .models import BlockCompletion
+
+class UnavailableCompletionData(Exception):
```

Comment:
Rather than adding the exception class here, please create a new file and add the new exception type into `completion/exceptions.py`